src/SFA.DAS.Employer.PR.Web/Views/DeclineCreateAccountConfirmation/Index.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿@using Microsoft.AspNetCore.Mvc.TagHelpers
+@model DeclineCreateAccountConfirmationViewModel
+
+@{
+    ViewData["Title"] = "Apprenticeship service account not created";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <div class="govuk-panel govuk-panel--confirmation">
+            <h1 class="govuk-panel__title">
+                Apprenticeship service account not created
+            </h1>
+        </div>
+        <div class="govuk-form-group">
+            <h2 class="govuk-heading-m govuk-!-margin-bottom-6 govuk-!-margin-top-6">
+                What happens next?
+            </h2>
+            <p class="govuk-body">
+                We’ve emailed @Model.ProviderName.ToUpper() to let them know you’ve declined their request to create an account.
```

Comment:
Since showing the name in upper case is a requirement, I would have a property in the view model so it can have a backing unit test.
`public string ProviderNameInUpperCase => ProviderName.ToUpper();`

Reply:
it is, this is not needed, am removing